### DISCUSSION TITLE
LoggedUser.CanManageShares

### DISCUSSION THREAD

**Discussion by AxisNL** [drakkan/sftpgo#842] [General]
Hi, I was looking at the user page, and I see a default user can manage shares and such, I want to disable that for my simple use case. But I can't find where.

In the templates, I see properties called LoggedUser.CanManageShares and LoggedUser.CanManageMFA I want to disable. But where can I disable these? 

When I look at the admin gui and I look at the user's properties, I can't seem to find these settings.

Angelo.

**Comment by drakkan** [comment]
Hi,

To disable the sharing feature open the web admin console, select your user and click on the edit icon, expand the "ACLs" section, and disable shares as you can see in the following screenshot:

![shares-disabled](https://user-images.githubusercontent.com/553263/168662381-69204640-8d5b-40a7-8e5f-8baba0bbb7bc.png)

save your user
